src/signals/incident/components/form/FileInput/FileInput.tsx

```diff
@@ -27,7 +32,7 @@ const FileInput = ({ handler, parent, meta }) => {
   )
 
   const checkNumberOfFiles = useCallback(
-    (file, index) => index < maxNumberOfFiles,
+    (_file, index) => index < maxNumberOfFiles,
```

Comment:
personally I would just use a lodash. " _ "
in some languages private methods are prefixed with an underscore, which might be a little confusing here.